### PR TITLE
Added avif support to blockreassurance icon images

### PR DESCRIPTION
**Pull request by buggyzap**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | I have added support to this new image format.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

Hi guys, I have simply added new .avif extension and image/avif mimetype to authorized.

### CODE REVIEW

File: controllers/admin/AdminBlockListingController.php [hunk 1]
```diff
@@ -145,8 +145,8 @@ public function displayAjaxSaveBlockContent()
         $type_link = (int) Tools::getValue('typelink');
         $id_cms = (int) Tools::getValue('id_cms');
         $psr_languages = (array) json_decode(Tools::getValue('lang_values'));
-        $authExtensions = ['gif', 'jpg', 'jpeg', 'jpe', 'png', 'svg'];
-        $authMimeType = ['image/gif', 'image/jpg', 'image/jpeg', 'image/pjpeg', 'image/png', 'image/x-png', 'image/svg', 'image/svg+xml'];
+        $authExtensions = ['gif', 'jpg', 'jpeg', 'jpe', 'png', 'svg', 'avif'];
+        $authMimeType = ['image/gif', 'image/jpg', 'image/jpeg', 'image/pjpeg', 'image/png', 'image/x-png', 'image/svg', 'image/svg+xml', 'image/avif'];
 
         if (!empty($picto) && !in_array(pathinfo($picto, PATHINFO_EXTENSION), $authExtensions)) {
             $errors[] = Context::getContext()->getTranslator()->trans('Image format not recognized, allowed formats are: .gif, .jpg, .png', [], 'Admin.Notifications.Error');
```
Comment (boherm) [inline controllers/admin/AdminBlockListingController.php]: Maybe, you can update this error message too!

**Review by boherm — CHANGES_REQUESTED** [review]
Hi @buggyzap, thanks for your contribution. Just a little comment :)

File: controllers/admin/AdminBlockListingController.php (hunk 1 above)
Comment (buggyzap) [reply controllers/admin/AdminBlockListingController.php]: ok ty

**Review by boherm — APPROVED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @buggyzap ,

Thanks for the PR ! 
Tested on 8.1.x and develop.
LGTM ✅ 

Could you add the extension on the help block as well please ?
Screenshot 2024-02-23 at 09 12 45

**Comment by buggyzap** [comment]
> Hello @buggyzap ,
> 
> Thanks for the PR ! Tested on 8.1.x and develop. LGTM ✅
> 
> Could you add the extension on the help block as well please ? Screenshot 2024-02-23 at 09 12 45

Ty I have added 1 more commit

**Review by florine2623 — APPROVED** [review]
Thanks @buggyzap ! LGTM ✅

**Comment by leemyongpakva** [comment]
Thanks @buggyzap

**Pull request merged**